api/client/client_test.go

```diff
@@ -577,6 +591,186 @@ func TestGetResources(t *testing.T) {
 	clt, err := srv.NewClient(ctx)
 	require.NoError(t, err)
 
+	t.Run("DatabaseServer", func(t *testing.T) {
```

Comment:
nit: any differences between these tests? maybe can have a generic helper function to avoid repeating.